reports/jshint-lines-of-shame.js

```diff
@@ -0,0 +1,48 @@
+// TODO: We need to figure out whether React will be supported natively for a reporter or if they
+// will be required to pre-compile
```

Comment:
Yea, I wrote straight jsx on line 44 for that reason but I'd imagine if this were a real reporter it would get fleshed out with some styling, maybe some containers. Who know. Pre-compilation of jsx for something more fully featured than this little list is NBD I just wanted to point out the things I found so we coud discuss.

![image](https://cloud.githubusercontent.com/assets/940667/7206598/e0455ec8-e4e7-11e4-814a-8a7877bf1e7a.png)